src/test/resources/tough.golden.vcf

```diff
@@ -16,7 +16,92 @@
 ##INFO=<ID=ReadPosRankSum,Number=1,Type=Float,Description="Z-score from Wilcoxon rank sum test of Alt vs. Ref read position bias">
 ##INFO=<ID=VQSLOD,Number=1,Type=Float,Description="Log odds ratio of being a true variant versus being false under the trained gaussian mixture model">
 ##INFO=<ID=culprit,Number=1,Type=String,Description="The annotation which was the worst performing in the Gaussian mixture model, likely the reason why the variant was filtered out">
-#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	tumor.chr20
+##contig=<ID=1,length=249250621,assembly=GRCh37>
```

Comment:
This predates this PR, but all 25 positions that [used to be checked in SomaticStandardRealDataSuite](https://github.com/hammerlab/guacamole/commit/6711f4adb203d96fa14ba7403b1a4ba68ad0ba37#diff-14a7269ee8be3937ec72f27c586af285L90) are here, as well as 9 more.